torch/csrc/jit/passes/quantization.cpp

```diff
@@ -1754,22 +1754,83 @@ std::tuple<at::Tensor, at::Tensor> FoldConvBatchNorm2dHelper::
   return std::make_tuple(new_w, new_b);
 }
 
+c10::optional<IValue> extractIValueFromGraph(
```

Comment:
nit: I think this should have a more explicit name, current name seems too general

Reply:
Sure. Will fix.